### PR TITLE
landing page and fixed routing

### PR DESCRIPTION
**Pull request by JosephL17**

### CODE REVIEW

**Review by Star-tide — CHANGES_REQUESTED** [review]
Need to revert navbar header zindex

File: frontend/src/components/Navbar.jsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ const navigation = [
 export default function Example() {
   const [mobileMenuOpen, setMobileMenuOpen] = useState(false)
   return (
-    <header className=" bg-primary">
```
Comment (Star-tide) [inline frontend/src/components/Navbar.jsx:11]: Need to change this back to bg-primary with NO zindex

**Review by Star-tide — CHANGES_REQUESTED** [review]
See comments for changes

**Review by Star-tide — APPROVED** [review]

**Review by polly-rado — APPROVED** [review]
accepting changes

**Pull request merged**